### PR TITLE
Add rubygem-journald-native package

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly

This is one of three new journal dependencies for foreman core.

---

### CODE REVIEW

File: rubygem-journald-native/rubygem-journald-native.spec [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+mkdir -p %{buildroot}%{gem_extdir_mri}
+cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+
+# Prevent dangling symlink in -debuginfo (rhbz#878863).
+rm -rf %{buildroot}%{gem_instdir}/ext/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/ext
```
Comment (lzap) [inline rubygem-journald-native/rubygem-journald-native.spec]: The only change I made so far was deleting this line:

    %{gem_instdir}/ext

**Comment by lzap** [comment]
I just pushed missing tags [test].

**Comment by ekohl** [comment]
[test packaging]

**Comment by ekohl** [comment]
Even though the build is green, we're still working on the refactoring of the job. It didn't actually build anything yet.

**Comment by ekohl** [comment]
Digging deeper: that's expected. Can you move it to the correct ```packages``` subdirectory?

**Comment by lzap** [comment]
Pushed, I am unsure about tags, shall I delete/push new tag into my repo?

**Comment by dLobatog** [comment]
http://koji.katello.org/koji/taskinfo?taskID=82538 - nightly

**Pull request merged**